.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             CURRDATE="$(date +%m/%d/%Y)"
 
             # Holiday dates specific to 2021 that will need to be updated next year
-            HOLIDAYS=("03/12/2021" "04/19/2021" "05/28/2021" "05/31/2021" "07/02/2021" "07/05/2021"
+            HOLIDAYS=("03/12/2021" "03/18/2021" "04/19/2021" "05/28/2021" "05/31/2021" "07/02/2021" "07/05/2021"
```

Comment:
@ehanna4 Did we not want to revert the addition of `03/18/2021` since it's probably not an annual holiday?

Reply:
Hi, I needed it to go to prod for testing purposes. Since it's in the list of 2021 specific Holidays it _technically_ doesn't even need to be reverted since after today it would have no effect, but for correctness I would like to remove it once I can test this.

Reply:
Ah right, sorry…I got confused about the testing of it again 🤦🏼